clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -8,7 +7,7 @@
 {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
 
-{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../emptydir/pod_with_emptydir.yaml"}}
+{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../volume-types/emptydir/pod_with_emptydir.yaml"}}
```

Comment:
Obviously this PR is going to break all the existing tests that are running.
Given they are experimental I'm fine with breaking them for a moment, but you should prepare a PR in test-infra that will fix the paths to configs and overrides so that we can merge that immediately after this PR will be merged.

Reply:
With https://github.com/kubernetes/test-infra/pull/13601 opened, I'm fine with approving this one.